examples/federation_authority/federation_authority/settingslocal.py.example

```diff
@@ -83,11 +83,12 @@ INSTALLED_APPS = [
     'django.contrib.messages',
     'django.contrib.staticfiles',
     # 'prettyjson',
-    "bootstrap_italia_template",
     'spid_cie_oidc.entity',
-    'spid_cie_oidc.authority',
     'spid_cie_oidc.onboarding',
-    'spid_cie_oidc.relying_party'
+    'spid_cie_oidc.relying_party',
+    'spid_cie_oidc.authority',
```

Comment:
I think that authority comes before onboarding and django bootstrap template too